CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Changelog
```

Comment:
Maybe we could add a first entry in the changelog mentioning the first version if the library with all the features we have implemented?